benchmarking/tpch/answers.py

```diff
@@ -660,11 +660,8 @@ def q22(get_df: GetDFFunc) -> DataFrame:
         res_1.where(col("C_ACCTBAL") > 0).agg(col("C_ACCTBAL").mean().alias("avg_acctbal")).with_column("lit", lit(1))
     )
 
-    res_3 = orders.select("O_CUSTKEY")
-
     daft_df = (
-        res_1.join(res_3, left_on="C_CUSTKEY", right_on="O_CUSTKEY", how="left")
-        .where(col("O_CUSTKEY").is_null())
+        res_1.join(orders.limit(1_000_000), left_on="C_CUSTKEY", right_on="O_CUSTKEY", how="anti")
```

Comment:
Why is there a limit here now?

Reply:
Oops good catch, forgot to remove them after testing the dedup benchmarking